src/components/pages/items/BestItemsSection.tsx

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from "react";
+import { useQuery } from "@tanstack/react-query";
+import { useMedia } from "@/store/MediaContext";
+import { getProducts } from "@/apis/product";
+import { GetProductsParams } from "@/types/api";
+import Item from "./Item";
+import styles from "./BestItemsSection.module.css";
+
+const pageSizeTable = { PC: 4, TABLET: 2, MOBILE: 1 };
```

Comment:
P2:
as const를 통해 더 정확한 타입을 추론해보세요~

```suggestion
const pageSizeTable = { PC: 4, TABLET: 2, MOBILE: 1 } as const;
```